spec/graphlient/static_client_query_spec.rb

```diff
@@ -15,6 +15,15 @@ module Graphlient::Client::Spec
         end
       end
 
+      StringQuery = Client.parse <<~GRAPHQL
+        query($some_id: Int) {
+          invoice(id: $some_id) {
+            id
+            feeInCents
+          }
+        }
+      GRAPHQL
+
       Query = Client.parse do
```

Comment:
Actually, I want separate `Query` constants since I have a test that checks that the parsed output of equivalent string-based and block-based queries is identical.

I did add additional `context` blocks as requested.